coderd/database/migrations/000098_template_version_variables.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS template_version_variables (
+    template_version_id uuid not null references template_versions (id) on delete cascade,
+    name text not null,
+    description text not null,
+    type text not null,
+    value text not null,
+    default_value text not null,
+    required boolean not null,
+    sensitive boolean not null,
+    unique (template_version_id, name)
+);
+
+COMMENT ON COLUMN template_version_variables.name IS 'Variable name';
+COMMENT ON COLUMN template_version_variables.description IS 'Variable description';
+COMMENT ON COLUMN template_version_variables.type IS 'Variable type';
+COMMENT ON COLUMN template_version_variables.value IS 'Variable value';
+COMMENT ON COLUMN template_version_variables.default_value IS 'Variable default value';
+COMMENT ON COLUMN template_version_variables.required IS 'Is variable required?';
+COMMENT ON COLUMN template_version_variables.sensitive IS 'Is variable sensitive?';
```

Comment:
I love that you added comments on these! ❤️ If you want to improve them slightly, they could explain more what it means. For example, sensitive means the value will be hidden in logs. I don't know if that's the case, just an idea.

Reply:
Fixed